src/parquet/parquet.h

```diff
@@ -40,6 +40,26 @@
 #include <thrift/protocol/TBinaryProtocol.h>
 #include <thrift/transport/TBufferTransports.h>
 
+//macros to disable padding. Required for Int96
```

Comment:
I already changed the code to sizeof(Int96) since padding is disabled in the current updated version. Once #22 and #23 are in, the same template code will work for Int96 as well since sizeof(Int96) will always be 12.
 If we plan to support the padding case as well, we have to use the pragmas you suggested to enable a specialization for Int96 where we explicitly set size to 12.